main/politeness_logi.R

```diff
@@ -0,0 +1,43 @@
+library(pscl)
+library(plyr)
+library(readr)
+
+# get data
+dat <- read.csv("/dev/stdin")
+dim(dat)
+# collect counts of marked politness words and store them in a file
+system2("src/convo_politeness",
+        stdout = "politeness_features.csv",
+        input = format_csv(dat))
+
+# redirect output to a file
+sink("politeness_logi.out")
+
+# read python's output from file
+dat <- read.csv("politeness_features.csv")
+m_pol <- glm(label ~
+              log(1 + HASHEDGE)
+            + (Please > 0)
+            + (Please_start > 0)
+            + (Indirect_.btw. > 0)
+            + Hedges
+            + (Factuality > 0)
+            + (Deference > 0)
+            + (Gratitude > 0)
+            + (Apologizing > 0)
+            + log(1 + X1st_person_pl.)
+            + log(1 + X1st_person)
+            + log(1 + X1st_person_start)
+            + log(1 + X2nd_person)
+            + (X2nd_person_start > 0)
+            + (Indirect_.greeting. > 0)
+            + (Direct_question > 0)
+            + log(1 + Direct_start)
+            + log(1 + HASPOSITIVE)
+            + log(1 + HASNEGATIVE)
+            + (SUBJUNCTIVE > 0)
+            + (INDICATIVE > 0)
+            , data = dat
+            , family = "binomial")
+summary(m_pol)
```

Comment:
don't you want to log this and the next statement? otherwise, generally noops

Reply:
These will go to the file I specified in `sink()`